README.md

```diff
@@ -71,79 +69,160 @@ resource "github_repository" "repository" {
 
 ## Examples
 
-For a complete example please see [examples](https://github.com/mineiros-io/terraform-github-team/tree/master/examples) directory.
+For a complete example please see [examples/] directory.
 
-## Makefile Targets
+## Module Argument Reference
 
-This repository comes with a handy
-[Makefile](https://github.com/mineiros-io/terraform-github-team/blob/master/Makefile).
-Run `make help` to see details on each available target.
+See [variables.tf] and [examples/] for details and use-cases.
 
-## Tests
+### Top-level Arguments
 
-This modules ships with a [Makefile](https://github.com/mineiros-io/terraform-github-team/blob/master/Makefile)
-that offers targets to execute the hooks and tests.
+#### Module Configuration
 
-**Execute all pre-commit hooks with Docker:**
+- **`module_depends_on`**: *(Optional `list(any)`)*
 
-```bash
-make docker/pre-commit-hooks
-```
+  A list of dependencies. Any object can be _assigned_ to this list to define a hidden external dependency.
 
-**Execute the Unit Test (deploy und undeploy the example):**
+#### Main Resource Configuration
 
-Please be aware, that in order to run the test you must provide GitHub credentials and an organization.
-Running the tests will create and destroy real resources.
+- **`name`**: *(Required `string`)*
 
-```bash
-GITHUB_ORGANIZATION=YOUR_GITHUB_ORGANIZATION \
-GITHUB_TOKEN=YOUR_GITHUB_TOKEN \
-make docker/unit-tests
-```
+  The name of the team.
+
+- **`push_repositories`**: *(Optional `set(string)`)*
+
+  A list of repository names the current team should get push (read-write) permission to.
+  Default is `[]`.
+
+- **`pull_repositories`**: *(Optional `set(string)`)*
+
+  A list of repository names the current team should get pull (read-only) permission to.
+  Default is `[]`.
+
+- **`maintainers`**: *(Optional `set(string)`)*
+
+  A list of users that will be added to the current team with maintainer permissions.
+  Default is `[]`.
+
+- **`members`**: *(Optional `set(string)`)*
+
+  A list of users that will be added to the current team with member permissions.
+  Default is `[]`.
+
+- **`admin_repositories`**: *(Optional `set(string)`)*
+
+  A list of repository names the current team should get admin (full) permission to.
+  Default is `[]`.
+
+- **`description`**: *(Optional `string`)*
+
+  A description of the team.
+  Default is `""`.
+
+- **`privacy`**: *(Optional `string`)*
+
+  The level of privacy for the team. Must be one of `secret` or `closed`.
+  Default is `""`.
```

Comment:
```suggestion
  Default is `""`.
```

Is this the correct default? If one of two values can only be set, what is the default for a newly created team?

Reply:
resolved in 6fb47ee